src/main/java/fi/otavanopisto/kuntaapi/server/index/AbstractIndexHander.java

```diff
@@ -0,0 +1,133 @@
+package fi.otavanopisto.kuntaapi.server.index;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import fi.otavanopisto.kuntaapi.server.integrations.KuntaApiConsts;
+import fi.otavanopisto.kuntaapi.server.settings.SystemSettingController;
+
+public abstract class AbstractIndexHander {
+
+  private static final String DEFAULT_INDEX = "kunta-api";
+  private static final String DEFAULT_CLUSTERNAME = "elasticsearch";
+  private static final String[] DEFAULT_HOSTS = new String[] {
+    "127.0.0.1:9300"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this IP "127.0.0.1" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1313)